### ISSUE TITLE
The status tool gives the wrong status (ru, branch 4.0)

### ISSUE THREAD

**Issue by eldarkg** [KiCad/kicad-i18n#103]
I use to translate the Poedit tool.
It show that all strings translated and no fuzzy.
But if I run the status tool it show me:
## ru
4894 translated messages
12 fuzzy translations
14 untranslated messages

**Comment by eldarkg** [comment]
Poedit 1.8.7.1, OS Ubuntu 16.04 LTS

**Comment by nickoe** [comment]
@eldarkg yeah, but if you open poedit on it after the tool has been run it will show the same. Can you share your poedit settings?

**Comment by nickoe** [comment]
If I look in the `ru/kicad.po` I see:
```
$ grep -r "Poedit-SearchPath" ru/
ru/kicad.po:"X-Poedit-SearchPath-0: 3d-viewer\n"
ru/kicad.po:"X-Poedit-SearchPath-1: bitmap2component\n"
ru/kicad.po:"X-Poedit-SearchPath-2: common\n"
ru/kicad.po:"X-Poedit-SearchPath-3: cvpcb\n"
ru/kicad.po:"X-Poedit-SearchPath-4: eeschema\n"
ru/kicad.po:"X-Poedit-SearchPath-5: gerbview\n"
ru/kicad.po:"X-Poedit-SearchPath-6: kicad\n"
ru/kicad.po:"X-Poedit-SearchPath-7: pagelayout_editor\n"
ru/kicad.po:"X-Poedit-SearchPath-8: pcb_calculator\n"
ru/kicad.po:"X-Poedit-SearchPath-9: pcbnew\n
```
and if I compare that with the POTDIRS file in the repo I see:
```
$ cat POTDIRS 
kicad
pcbnew
common
eeschema
cvpcb
3d-viewer
gerbview
bitmap2component
new
polygon
pcb_calculator
potrace
tools
pagelayout_editor
include
lib_dxf
```
I guess that is the reason for the mismatch?

**Comment by eldarkg** [comment]
Fixed in 23e033a
